vhost-user-backend/Cargo.toml

```diff
@@ -17,6 +17,7 @@ gpu-socket = ["vhost/gpu-socket"]
 libc = "0.2.39"
 log = "0.4.17"
 userfaultfd = { version = "0.8.1", optional = true }
+uuid = { version = "1.8.0", features=["v4", "fast-rng", "macro-diagnostics"] }
```

Comment:
Can you explain why we need these 3 features enabled in the commit description?

Reply:
IIUC this is used just for testing, so it should go in `dev-dependencies` section if it's the case.

Reply:
> Can you explain why we need these 3 features enabled in the commit description?

we need just one `v4`, thanks for pointing that out, I will remove the other features.

> IIUC this is used just for testing, so it should go in dev-dependencies section if it's the case.

Ack